content/en/blog/2024/trustee-deployment.md

```diff
@@ -0,0 +1,351 @@
+```
+
+{{% alert color="info" %}}
+For more information on the Operator Lifecycle Manager (OLM) and the operator installation procedure from OperatorHub.io, please consult this [guide](https://operatorhub.io/how-to-install-an-operator).
+{{% /alert %}}
+
+
+### Namespace creation
+
+This is the default Namespace, where all the relevant Trustee objects will be created.
+
+```bash
+kubectl apply -f - << EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kbs-operator-system
+EOF
+```
+
+### Operator Group
+
+An Operator group, defined by the OperatorGroup resource, provides multi-tenant configuration to OLM-installed Operators:
+
+```bash
+kubectl apply -f - << EOF
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: kbs-operator-system
+  namespace: kbs-operator-system
+spec:
+EOF
+```
+
+### Subscription
+
+A subscription, defined by a Subscription object, represents an intention to install an Operator. It is the custom resource that relates an Operator to a catalog source:
+
+```bash
+kubectl apply -f - << EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: kbs-operator-system
+  namespace: kbs-operator-system
+spec:
+  channel: alpha
+  installPlanApproval: Automatic
+  name: trustee-operator
+  source: operatorhubio-catalog
+  sourceNamespace: olm
+  startingCSV: trustee-operator.v0.1.0
+EOF
+```
+
+{{% alert color="info" %}}
+Note that the Trustee operator has been already published in the operator [hub catalog](https://operatorhub.io/).
+{{% /alert %}}
+
+### Check Trustee Operator installation
+
+Now it is time to check if the Trustee operator has been installed properly, by running the command:
+
+```bash
+kubectl get csv -n kbs-operator-system
+```
+
+You should expect something like:
+
+```bash
+NAME                      DISPLAY            VERSION   REPLACES   PHASE
+trustee-operator.v0.1.0   Trustee Operator   0.1.0                Succeeded
+```
+
+## Configuration
+
+The Trustee Operator configuration requires a few steps. Some of the steps are provided as an example, but you may want to customize the examples for your real requirements.
+
+### Authorization key-pair generation
+
+First of all, you’d need to create the key pairs for Trustee authorization. The *publicKey* is used by Trustee for client authorization, the *private
+Key* is used by the client to prove its identity and register keys/secrets.
+
+
+Create secret for client authorization:
+
+```bash
+openssl genpkey -algorithm ed25519 > privateKey
+openssl pkey -in privateKey -pubout -out publicKey
+kubectl create secret generic kbs-auth-public-key --from-file=publicKey -n kbs-operator-system
+```
+
+### HTTPS configuration
+
+It is recommended to enable the HTTPS protocol for securing the server API. In this example we're going to create a self-signed certificate using the following template:
```

Comment:
I think part of the reason for choosing HTTPS is also to bind the TEE to a given KBS by seeding the pubkey/cert to a TEE (as measured init data).